.tekton/tasks/send-slack-notification.yaml

```diff
@@ -0,0 +1,66 @@
+apiVersion: tekton.dev/v1beta1
+kind: Task
+metadata:
+  name: send-slack-notification
+spec:
+  description: >-
+    These tasks post a rhtap staging e2e ci run status message to a Slack channel.
```

Comment:
Do you think that in the future we can improve this task by linking `@handles` to specific components so that a specific team is tagged/alerted when the error is raised?

You can close this thread after you answered.

Reply:
yes, we can do that. For example I am adding `@ic-appstudio-qe` [here](https://github.com/redhat-appstudio/infra-deployments/pull/2383/files/e3c2fb3f6925cc7aa1db36257749512848036bf9#diff-fbe4942329f5fb064239899264119e14b89b72b3a708eea8d5a6ff686a584a9eR48) when the e2e test fails on staging. 
Similarly we can do it for team specific.